main.c

```diff
@@ -20,6 +20,7 @@
 #include "nsxiv.h"
 #include "commands.h"
 #define _MAPPINGS_CONFIG
+#define _IMAGE_CONFIG
```

Comment:
I would avoid defining _IMAGE_CONFIG in 2 files; it is a code smell as it'll cause the relevant config files to be defined twice. 

Not sure if you should exactly where nav_button and NAV_WIDTH should go... I feel like they relate more to the window code since than image.

Reply:
I see, you mean the other `*_CONFIG` are defined in image.c, window.c and thumbs.c. What should I do instead? There will be the same issue if it is in `_WINDOW_CONFIG`.

Reply:
That would work along width moving NAV_WIDTH to be defined in the _WINDOW_CONFIG section in config.h

Reply:
One problem, the function is used in `reset_cursor` function in this file:
https://github.com/nsxiv/nsxiv/pull/155/files#diff-a0cb465674c1b01a07d361f25a0ef2b0214b7dfe9412b7777f89add956da10ecL434

Reply:
I haven't taken a close look into this, but couldn't `NAV_WIDTH` be defined under _MAPPINGS_CONFIG ?

Reply:
Or just declare the function for reset_cursor. Or put it in a header since it is also used in commands.c

Reply:
I should probably be defined under MAPPING anyways, since it relates more to mapping than IMAGE or WINDOW

Reply:
I agree with @N-R-K, window.c seems mostly related to window UI elements.